pulsar-client-tools/src/test/java/org/apache/pulsar/admin/cli/TestRunMain.java

```diff
@@ -31,7 +31,7 @@
     public void runMainNoArguments() throws Exception {
         PulsarAdminTool.setAllowSystemExit(false);
         PulsarAdminTool.main(new String[0]);
-        assertEquals(PulsarAdminTool.getLastExitCode(), 1);
+        assertEquals(PulsarAdminTool.getLastExitCode(), 0);
```

Comment:
This change looks unrelated

Reply:
yes, it was for testing. created a separate PR: https://github.com/apache/pulsar/pull/12675